packages/react/src/components/Link/Link.tsx

```diff
@@ -92,7 +95,6 @@ const Link = React.forwardRef<HTMLAnchorElement, PropsWithChildren<LinkProps>>(
     });
     const rel = target === '_blank' ? 'noopener' : undefined;
     const linkProps: AnchorHTMLAttributes<HTMLAnchorElement> = {
-      className,
```

Comment:
Do we want to remove this?

Reply:
@tw15egan  I previously had move down the `className` to the component itself. But it also work as it was if I do an ternary if.
I moved it back

Reply:
Oh that makes sense, I'm fine either way, I think we just need to set it to `null` rather than `` because I was seeing this in the markup: 

<img width="275" alt="Screenshot 2023-11-21 at 12 50 28 PM" src="https://github.com/carbon-design-system/carbon/assets/11928039/1de006a3-1e1d-4165-be23-b1ec1433963f">

Reply:
Fixed it!